nixos/doc/manual/release-notes/rl-2411.section.md

```diff
@@ -87,6 +87,8 @@
 
 - [Goatcounter](https://www.goatcounter.com/), an easy web analytics platform with no tracking of personal data. Available as [services.goatcounter](options.html#opt-services.goatcocunter.enable).
 
+- [Porn-Vault](https://gitlab.com/porn-vault/porn-vault), a porn organizing web app. Available as [services.porn-vault](#opt-services.porn-vault.enable).
```

Comment:
One thing, I am the last person to judge anyone but considering this will show in the homepage for 25.05 I think it is better that we don't add release notes for this module.

Everyone that needs this will probably find it, so I don't think we need to add it here.

Reply:
Idk lmao, the guidelines say I should add it there so I added
But if y’all need me to remove it I can definitely remove it lol